src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -90,7 +102,7 @@ For example, to remove one node in a three-node cluster, first add a new node to
 
 Complete the following prerequisites before you remove one or more nodes from a Rook Ceph cluster:
 
-* (Recommended) Upgrade Rook Ceph to v1.4 or later.
+* Upgrade Rook Ceph to v1.4 or later. Attempting to remove a node from a cluster that uses a Rook Ceph version earlier than v1.4 can cause Ceph to enter an unhealthy state. For example, see [Rook Ceph v1.0.4 is Unhealthy with Mon Pods Not Rescheduled](#rook-ceph-v104-is-unhealthy-with-mon-pods-not-rescheduled) under _Troubleshoot Node Removal_ below.
```

Comment:
Could we also supplement here saying that ideally we recommend use and keep updated with the latest Rook version provided?

Reply:
Also, wdyt about we link https://rook.io/docs/rook/latest/Getting-Started/release-cycle/ and say that most recent two minor Rook releases are those that are actively maintained by Rook itself.

Reply:
Great idea!